tensorflow_hub/resolver.py

```diff
@@ -534,3 +535,10 @@ def _call_urlopen(self, request):
 
   def is_http_protocol(self, handle):
     return handle.startswith(("http://", "https://"))
+
+  def _maybe_disable_cert_validation(self):
```

Comment:
Let's add a test to resolver_test.py (similar to testCacheDir()) that would exercise setting the new environment variable. We won't be able to truly test the logic in a unittest setting but at least we can make sure that if the condition is triggered the logic doesn't break.

Reply:
I do not know how to do this (where to add, what to test, etc). Can you please provide me with a guideline on what I should test.

Reply:
We can add the follow test 

```python
  def testGetModulePathTarGz_NoCert(self):
   os.environ[resolver. _TFHUB_DISABLE_CERT_VALIDATION] = "true"
    FLAGS.tfhub_cache_dir = os.path.join(self.get_temp_dir(), "cache_dir")
    http_resolver = compressed_module_resolver.HttpCompressedFileResolver()
    path = http_resolver(self.module_handle)
    files = os.listdir(path)
    self.assertListEqual(sorted(files), ["file1", "file2", "file3"])
```

to compressed_modules_resolver_test.py.

The code could be tested with 

```shell
$ bazel test tensorflow_hub:compressed_module_resolver_test
```

It's a shallow test but at least it makes sure that there are no typos in _maybe_disable_cert_validation()